pkg/reconciler/autoscaling/versionwrapper/controller.go

```diff
@@ -0,0 +1,26 @@
+package versionwrapper
```

Comment:
For SO operator we have code that actually tweaks the manifests (between v2 and v2b2), right?

But here the wrapper is needed because the actual Go API is invoked, which is the difference to eventing, right?

Reply:
Yes and yes.